test/internal/opts/process_compiler_opts_tests.bzl

```diff
@@ -317,33 +284,6 @@ def process_compiler_opts_test_suite(name):
         expected_build_settings = {
             "ENABLE_TESTABILITY": "True",
         },
-        expected_swiftcopts = [
-            "-F__BAZEL_XCODE_DEVELOPER_DIR__/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks",
-            "-F__BAZEL_XCODE_SDKROOT__/Developer/Library/Frameworks",
-            "-I__BAZEL_XCODE_DEVELOPER_DIR__/Platforms/iPhoneSimulator.platform/Developer/usr/lib",
-            "-DDEBUG",
-            "-application-extension",
-            "weird",
-            "-Xcc",
-            "-iquote$(PROJECT_DIR)",
-            "-Xcc",
-            "-iquote$(PROJECT_DIR)/bazel-out/ios-sim_arm64-min15.0-applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin",
-            "-unhandled",
-            "-Xcc",
-            "-fmodule-map-file=/abs/path",
-            "-Xcc",
-            "-I/abs/path",
-            "-Xcc",
-            "-iquote/abs/path",
-            "-Xcc",
-            "-isystem/abs/path",
-            "-Xcc",
-            "-O0",
-            "-Xcc",
-            "-DDEBUG=1",
-            "-Xcc",
-            "-Fsomewhere",
-        ],
```

Comment:
I'm sad we are losing these, but I'll add tests to the execution phase processing tool to make up for it.